handlers/create_lease.go

```diff
@@ -25,7 +26,7 @@ func (c createLeaseReq) maxTimeDur() time.Duration {
 }
 
 // CreateLease creates the handler that responds to the POST /lease endpoint
-func CreateLease(containerService *container.Service) http.Handler {
+func CreateLease(containerService *container.Service, services k8s.ServiceInterface) http.Handler {
```

Comment:
what is the handler going to do with these `containerService` and `k8s.ServiceInterface` params?